api/v6/report_server.thrift

```diff
@@ -359,6 +359,11 @@ service codeCheckerDBAccess {
                                                           2: Encoding encoding)
                                                           throws (1: shared.RequestFailed requestError),
 
+  // Return true if review status change is disabled.
+  // PERMISSION: PRODUCT_ACCESS or PRODUCT_STORE
+  bool reviewStatusChangeIsDisabled()
```

Comment:
Maybe it would be better to name it like this `isReviewStatusChangeDisabled` for the variables and api calls.